WBooks/Colors/Colors.swift

```diff
@@ -17,4 +18,11 @@ extension UIColor {
     static var wBooksBackground: UIColor {
         return UIColor(red: 236/255, green: 246/255, blue: 249/255, alpha: 1)
     }
+    
+    static func wBooksButtonGradient() -> ViewGradient {
```

Comment:
This should be in `UIButton` extension... create a function that sets this ViewGradient to buttons.gradient and in button's `didSet` call it